polaris-migrator/src/migrations/styles-replace-inline-comments/transform.ts

```diff
@@ -0,0 +1,37 @@
+import type {API, FileInfo, Options} from 'jscodeshift';
+import type {Plugin} from 'postcss';
+import postcss from 'postcss';
+import stylelint from 'stylelint';
+
+const plugin = (): Plugin => {
+  return {
+    postcssPlugin: 'styles-replace-inline-comments',
+    Comment(comment) {
+      comment.raws.inline = false;
+      comment.raws.right = ' ';
+      comment.raws.left = ' ';
+    },
+  };
+};
+
+export default async function transformer(
+  file: FileInfo,
+  _: API,
+  options: Options,
+) {
+  return postcss([
+    stylelint({
+      config: {
+        extends: [options.config ?? '@shopify/stylelint-polaris'],
```

Comment:
Can you help me understand why we're using `stylelint` and `@shopify/stylelint-polaris` in this migration? We don't appear to be operating on the `stylelint` results in the migration plugin. I'm curious if we can follow the boilerplate Sass migration pattern ([for example](https://github.com/Shopify/polaris/blob/eb93b7f1ccfd3a3778860b50de95465147207fab/polaris-migrator/src/migrations/styles-replace-custom-property/transform.ts#L33-L40))